CHANGES.md

```diff
@@ -1,2 +1,6 @@
+# 0.0.3 ....
+
+- javadoc on configuration attributes (IDE tooltips)
```

Comment:
I think this would benefit from an `[IMPROVED]` tag - I'll add it in after merging.